ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -180,6 +193,20 @@ const GENE_DISEASE_DETAIL_SECTIONS = [
     showDetails: gene => gene.genCc?.classifications,
     detailsDisplay: gene => (<GenCC genCc={gene.genCc} />),
   },
+  {
+    color: 'purple',
+    description: 'ClinGen Dosage Sensitivity',
+    label: 'ClinGen',
+    showDetails: gene => gene.clinGen,
+    detailsDisplay: gene => (
+      <NoBorderTable basic="very" compact="very">
+        {gene.clinGen.haploinsufficiency &&
+          <ClinGenRow value={gene.clinGen.haploinsufficiency} href={gene.clinGen.href} label="Haploinsufficiency" />}
+        {gene.clinGen.triplosensitivity &&
+          <ClinGenRow value={gene.clinGen.triplosensitivity} href={gene.clinGen.href} label="Triplosensitivity" />}
```

Comment:
Does it need protection when `gene.clinGen` is undefined?

Reply:
No the `showDetails` condition makes sure this only renders when it is defined